apps/profiles/views.py

```diff
@@ -587,3 +587,10 @@ def list(self, request, format=None):
         user = request.user
         serializer = ProfileSerializer(user)
         return response.Response(serializer.data)
+
+    def put(self, request, pk=None):
+        user = request.user
+        serializer = ProfileSerializer(user, data=request.data, partial=True)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save(user=request.user)
```

Comment:
`user` is already defined as a variable, could use that instead of `request.user`?